controllers/utils/utils.go

```diff
@@ -213,8 +212,8 @@ func GetIstioControlPlaneName(ctx context.Context, cli client.Client) (istioCont
 					meshNamespace = namespace
 				} else {
 					log.V(1).Info("Mesh Namespace is not set in DSCI")
-					// at this point, it is not set anywhere, lets just use the default
-					meshNamespace = constants.IstioNamespace
+					// at this point, it is not set anywhere, lets return an error
```

Comment:
it is better to return the default instead panicking, no @israel-hdez ?

Reply:
My rationale here is that we shouldn't be using any defaults in the code. We should be forcing this to be specified.
Hoping I'm not wrong, it is always going to be set on the Deployment. If it is not set, the manifests would be faulty.

Reply:
That make sense, but is this the kind of error we want to block the operator startup?

Reply:
As it is, this should not block operator startup. Although, it will crash during reconciliation of some ISVC.
I expect no crash happening as long as all configuration is correct.

But yes, we can block operator startup to fail early. The panics would work as a back-up. Probably better to do that in another PR to not complicate this one.

Reply:
Right, @mholder6 can you please open a follow-up Jira?